vision_stack/buoy_detect/README.md

```diff
@@ -0,0 +1,24 @@
+BUOY_DETECT
+-----------
+This repository contains the codes that uses ML (Machine Learning) techniques to detect the buoy. This is a combination of 3 parts - 
+<br>
```

Comment:
@athithyakumar : Fix the problems in this file as well.

@pranaypratyush You can't tag him because he isn't in the organisation yet. I will add him now.